src/app/views/assignments/popups/test-live/test-live.component.ts

```diff
@@ -169,7 +172,7 @@ export class TestLiveComponent implements OnInit {
 
   async setQuestionAsMarkerdForReview() {
     const quesForMarkedAsReview = new QuestionMarkedForReviewModel();
-    quesForMarkedAsReview.answerText = null;
+    quesForMarkedAsReview.answerText = this.question.name;
```

Comment:
Answer text should not be question name. It should be the answer in the TITA box if its TITA